samples/fr-platform/idm/conf/authentication.json

```diff
@@ -17,18 +17,30 @@
         },
         "authModules" : [
             {
-                "name": "CLIENT_CERT",
-                "properties": {
-                    "queryOnResource": "endpoint/static/user",
-                    "defaultUserRoles": [
-                        "internal/role/openidm-cert"
-                    ],
-                    "allowedAuthenticationIdPatterns": [
-                        "CN=ig, O=forgerock"
+                "name" : "STATIC_USER",
+                "properties" : {
+                    "queryOnResource" : "endpoint/static/user",
+                    "username" : "anonymous",
+                    "password" : "&{openidm.anonymous.password}",
+                    "defaultUserRoles" : [
+                        "internal/role/openidm-reg"
+                    ]
+                },
+                "enabled" : true
+            },
+            {
+                "name" : "STATIC_USER",
+                "properties" : {
+                    "queryOnResource" : "endpoint/static/user",
+                    "username" : "&{ig.client.username}",
+                    "password" : "&{ig.client.password}",
```

Comment:
I gather these are the creds that IG needs to query IDM - hence the need to set the password here.

Reply:
Yep